### DISCUSSION TITLE
Is this Abandonware??

### DISCUSSION THREAD

**Discussion by wgcsteve** [fullstackhero/blazor-starter-kit#463] [General]
Is anyone actively working on this project?   It contains known vulnerabilities!
We have used this template on a few projects in the past, but we might need to find something more current.

**Comment by pratikghag1089** [comment]
Last release was on 2021.... Today is 2024.

**Reply by wgcsteve** [reply]
That wasn't very helpful.  

There are pull requests from as recent as November 2023.  Trying to see if anyone has forked it, or if there was a branch that is more current.

**Reply by pratikghag1089** [reply]
Alright. Just wondering, have you used this https://github.com/oqtane/oqtane.framework?

**Comment by RoLYroLLs** [comment]
Just as I was about to try this out...

**Reply by neozhu** [reply]
I'm still going on my blazor server project. https://github.com/neozhu/CleanArchitectureWithBlazorServer

**Reply by RoLYroLLs** [reply]
Nice! I'll look into it!

**Comment by wgcsteve** [comment]
FYI, we were able to update everything to .Net 8 and update all of the components.  So we are continuing to use this template.  

We wanted to submit a pull request, but the author has completely abandoned the project.  If we find time, maybe we'll create a fork.

Here are the high-level steps if you were so interested:

```
Basic steps to upgrade Blazor Hero from net core 5.0 to 8.0:
1. Download .Net Upgrade Assistant extension and upgrade all projects net core version.
2. Update appsettings.json file in Server project, AppConfiguration.Secret entry should have a length of 32 (now it has 26)
3. Update appsettings.json file in Server project, add TrustServerCertificate=True in connection string
4. Upgrade Automapper version to 11.0.0 in projects Application, Infraestructure and ClientInfraestructure
5. Remove AutoMapper.Extensions.Microsoft.DependencyInjection dependendy from projects Application, Infraestructure and ClientInfraestructure (edited) 
```

Then you'll also want to update MudBlazor to the latest.

I'll post again if we ever create a forked project.
